src/pystow/utils.py

```diff
@@ -928,6 +928,11 @@ def path_to_sqlite(path: Union[str, Path]) -> str:
     return f"sqlite:///{path.as_posix()}"
 
 
-def gunzip(source, target) -> None:
-    """Unzip a file in the source to the target."""
-    raise NotImplementedError
+def gunzip(source: Union[str, Path], target: Union[str, Path]) -> None:
+    """Unzip a file in the source to the target.
+
+    :param source: The path to an input file
+    :param target: The path to an output file
+    """
+    with gzip.open(source, "rb") as in_file, open(target, "wb") as out_file:
+        out_file.write(in_file.read())
```

Comment:
```suggestion
        shutil.copyfileobj(in_file, out_file)
```
This avoids reading the whole file into memory (cf. [here](https://docs.python.org/3/library/shutil.html#shutil.copyfileobj))

Reply:
hella